src/app/search/_components/NoResult.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames/bind';
+import styles from './NoResult.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface NoResultProps {
+  keyword: string;
+}
+
+export default function NoResult({ keyword }: NoResultProps) {
+  return (
+    <div className={cn('wrapper')}>
+      <div className={cn('title-wrapper')}>
+        <span className={cn('keyword')}>&apos;{keyword}&apos; </span>
```

Comment:
```suggestion
       <span className={cn('keyword')}>{`'${keyword}'`}</span>
```
사소하지만 이 방법이 조금 더 모던해보이는군요